.github/workflows/pipeline-segment-electron-test.yml

```diff
@@ -40,6 +40,7 @@ env:
   CHROMIUM_GIT_COOKIE_WINDOWS_STRING: ${{ secrets.CHROMIUM_GIT_COOKIE_WINDOWS_STRING }}
   ELECTRON_OUT_DIR: Default
   ELECTRON_RBE_JWT: ${{ secrets.ELECTRON_RBE_JWT }}
+  ACTIONS_STEP_DEBUG: ${{ secrets.ACTIONS_STEP_DEBUG }}
```

Comment:
This is going to enable debugging for **all** pipelines, instead of just one. I think this secret should be a branch name or something so that the check is `current_branch == secrets.debug_branch_name` or something.